src/languageTools/PathConverters.js

```diff
@@ -36,7 +36,7 @@ define(function (require, exports, module) {
 
         let filePath = decodeURIComponent(url.pathname);
         if (brackets.platform === 'win') {
-            if (filePath[0] === '/') {
+            if (filePath && filePath.includes(":/") && filePath[0] === '/') {
```

Comment:
Can you please describe the format of paths that are returned by the server in different scenarios?

That would help assess this change.

Reply:
@shubhsnov 
Below are the file format in different scenario
For local disk file: file:///C:/<file_path>
For Network FIle : file:////blrstore.corp.adobe.com/<file_path>
For Temp New File : file:///_brackets1234/<file_name>